app/helpers/sufia_helper.rb

```diff
@@ -15,4 +15,22 @@ def institution_name
   def institution_name_full
     Site.institution_name_full || super
   end
+
+  def displayable_content_block(content_block, class_string = '', id_string = '')
+    return if content_block.value.blank?
+    raw("<div id='#{id_string}' class='#{class_string}'>#{content_block.value}</div>")
```

Comment:
Instead of passing the class and id, how about we pass `options: {}` and pass it through to the content_tag?